external-crates/move/crates/move-unit-test/src/lib.rs

```diff
@@ -150,14 +150,19 @@ impl UnitTestingConfig {
                 .set_flags(flags)
                 .run::<PASS_CFGIR>()
                 .unwrap();
-        let (_, compiler) =
-            diagnostics::unwrap_or_report_diagnostics(&files, comments_and_compiler_res);
+        let (_, compiler) = diagnostics::unwrap_or_report_diagnostics(
```

Comment:
Would it be worth writing a variant of this that takes the pass/diag pair instead of calling `map_err` so many places?